contracts/gateway/AxelarAmplifierGatewayProxy.sol

```diff
@@ -14,4 +16,8 @@ contract AxelarAmplifierGatewayProxy is Proxy {
     function contractId() internal pure override returns (bytes32) {
         return keccak256('axelar-amplifier-gateway');
     }
+
+    function getContractId() public pure returns (bytes32) {
+        return CONTRACT_ID;
+    }
```

Comment:
```suggestion
        return CONTRACT_ID;
    }
```
contractId is already exposed via the implementation, so exposing a different value via another getter can be an issue